recipes/spdlog/all/conanfile.py

```diff
@@ -51,7 +51,9 @@ def configure(self):
             raise ConanInvalidConfiguration("Visual Studio build for shared library with MT runtime is not supported")
 
     def requirements(self):
-        if Version(self.version) >= "1.7.0":
+        if Version(self.version) >= "1.9.0":
```

Comment:
It is always possible to override `fmt/7.1.3` with a newer version, and make the build fail.
Do we need to check in `validate` that the dependency `fmt` has a valid version?

Reply:
That's a good idea.

Reply:
you can't check versions of dependencies in `validate()`

Reply:
Thanks. There was already a check in build.